docs/contents/dev-docs/testing.md

```diff
@@ -57,13 +57,17 @@ If you already set this environment variable above during build stage, ignore th
 SERVICE=ecr
 ```
 
-To generate temporary credentials for functional testing, pass in a role ARN. This will create a resource in your account in us-west-2 region
-for the respective AWS Service:
+To generate temporary credentials for functional testing, pass in a role ARN. 
 
 ```
-AWS_ROLE_ARN=arn:aws:iam::12345678980:role/Admin-k8
+AWS_ROLE_ARN=arn:aws:iam::<account-id>:role/<role-name>
 ```
 
+!!! warning
+    The role ARN passed should exist in your AWS account, and has necessary permissions to create/describe/delete the resource. 
```

Comment:
The role *should* or *must* exist? What happens if it doesn't? Can you also add an example here, as in: if you want to be test read-only from an S3 bucket then use the ARN ...

Reply:
Good point. Role must exist and have necessary permissions. I will go ahead and add some examples roles. We cannot have a read only role unfortunately as we are create a resource and describe it to make sure it is successfully created as part of E2E tests.